DirectEmployers/MyJobs
# Changes to "Distribution Status" switch [PD-3430]

## What's this PR do?
- Hides switch if the user is viewing a single organization
- Adds a confirm when disabling the switch
## Screenshots (if appropriate)
![2017-07-07-145104_819x432_scrot](https://user-images.githubusercontent.com/3379265/27972283-cce7331c-6323-11e7-99be-314bb75e3bf0.png)
## Where should the reviewer start?

## How should this be manually tested?
Select an organization. In any of that organization's subscreens, the switch should not be present.
Head back to your main TJD screen. Switch off distribution. You should see the pictured dialog.
## Any background context you want to provide?
The text is not my own. If anyone has suggestions, I'm all ears.
